apps/clearance_ui/src/styles/globals.css

```diff
@@ -97,4 +97,24 @@
     .logo-brackets {
         color: #ff3366 !important;
     }
+
+    .color-94C9A9 {
+        color: #c6caed;
+    }
```

Comment:
The name of the color does not match the value.

More generally, arbitrary values can be directly used with https://tailwindcss.com/docs/customizing-colors#arbitrary-values.